cinema_service/settings.py

```diff
@@ -134,7 +146,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
Ensure that the `MEDIA_ROOT` directory exists and is writable by the application to avoid issues with media file uploads.